lib/src/common/orgs_service.dart

```diff
@@ -167,6 +175,29 @@ class OrganizationsService extends Service {
     return completer.future;
   }
 
+  /// Returns the membership status for a user in a team given the [orgName] and [teamName].
+  Future<TeamMembershipState> getTeamMembershipByName(
+      String orgName, String teamName, String user) {
+    final completer = Completer<TeamMembershipState>();
+
+    github
+        .getJSON(
+          '/orgs/$orgName/teams/$teamName/memberships/$user',
+          statusCode: 200,
+          fail: (http.Response response) {
+            if (response.statusCode == 404) {
```

Comment:
Use `NOT_FOUND` to make it easier to read

https://github.com/SpinlockLabs/github.dart/blob/11a83b4fc9558b7f8c47fdced01c7a8dac3c65b2/lib/src/common/util/utils.dart#L84

Reply:
Ah okay. I used HttpStatus but I will change it to these constants.